src/Message/ZipFile.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Message;
+
+/**
+ * The message for Zip file handler.
+ */
+class ZipFile
```

Comment:
I think we should change to name to reflect what it's doing. According to the Symfony message methodology, in the command message bus there is one message (command) and one handler. And this message doesn't just zip all files, or any file, it's very specific. it zips all dataset files. Maybe call it: zipAllDatasetFiles? We might get a generic zipFile message some day.

Reply:
Yes, I think so too and it is confusing as the Util class name is also similar